docs/xls-65d-single-asset-vault/reference/transactions/vault-withdraw.md

```diff
@@ -0,0 +1,70 @@
+---
+seo:
+    description: Withdraws a specified number of assets from a vault in exchange for shares.
+labels:
+  - Transactions
+  - Single Asset Vault
+---
+
+# VaultWithdraw
+
+[[Source]](https://github.com/XRPLF/rippled/blob/9d619b9dc579c592f0560c1b40fd3c98d7587d23/src/xrpld/app/tx/detail/VaultWithdraw.cpp "Source")
+
+Withdraws assets from a vault or redeems vault shares for assets. The number of shares burned or assets received depends on the [exchange rate](../../concepts/single-asset-vault.md#exchange-algorithm), which adjusts based on the vault’s total assets and any [unrealized losses](../../concepts/single-asset-vault.md#paper-loss-unrealized-loss).
+
+{% admonition type="info" name="Note" %}
+The `VaultWithdraw` transaction does not respect the Permissioned Domain rules. In other words, any account that holds the shares of the vault can withdraw them. This is to avoid a situation where a depositor deposits assets to a private vault to then have their access revoked by invalidating their credentials, and thus loosing access to their funds.
+{% /admonition %}
+
+## Example {% $frontmatter.seo.title %} JSON
+
+```json
+{
+  "TransactionType": "VaultWithdraw",
+  "Account": "ra5nK24KXen9AHvsdFTKHSANinZseWnPcX",
+  "Fee": "12",
+  "Flags": 0,
+  "LastLedgerSequence": 7108682,
+  "Sequence": 8,
+  "VaultID": "77D6234D074E505024D39C04C3F262997B773719AB29ACFA83119E4210328776",
+  "Amount" : "10000",
+  "Destination": "ruazs5h1qEsqpke88pcqnaseXdm6od2xc"
+}
+```
+
+## {% $frontmatter.seo.title %} Fields
+
+In addition to the [common fields](https://xrpl.org/docs/references/protocol/transactions/common-fields#transaction-common-fields), {% code-page-name /%} transactions use the following fields:
+
+| Field Name              | JSON Type     | Internal Type | Required? | Description         |
+| :-----------------------| :------------ | :------------ | :-------- | :-------------------|
+| `VaultID`               | String        | Hash256       | Yes       | The unique identifier of the vault to which the assets are deposited. |
+| `Amount`                | Number        | STAmount      | Yes       | The exact amount of vault asset to withdraw or vault share to redeem. |
+| `Destination`           | String        | AccountID     | No        | An account to receive the assets. This account must be able to receive the vault asset or the transaction fails.                   |
+
+There are two ways to specify the transaction `Amount` field:
+
+| Withdraw | Redeem |
```

Comment:
```suggestion
| Specify Assets | Specify Shares |
```
This is maybe nitpicky, but redeeming feels like the more accurate way to explain what's happening with the shares and assets. You're never withdrawing without redeeming.